src/input/code_templates/c/static_code/exi_basetypes.c.jinja

```diff
@@ -134,6 +134,24 @@ int exi_basetypes_convert_64_from_signed(const exi_signed_t* exi_signed, int64_t
     return res;
 }
 
+static void _reverse_array(uint8_t* data, size_t data_size)
```

Comment:
I get why we want to reverse it, but I rather see it written backwards from the get go to speed it up a bit.

Reply:
Yes, EXI internally uses the other direction, from LSB to MSB. I took a "sort of" internal vote on this:
- MSB to LSB seemed more natural to two people (YMMV though).
- OpenV2G also does a byte order reversal.

You mean by "backwards from the get" to fill the array in the opposite order, instead of reversing it after the fact. (Speaking for the decoding direction only here, obviously.)

Let me have a look at that. In the decoder, we grow the result array from the inputs. I don't us gaining much from creating a larger array, filling it from the back, and then shrinking it to size.

From the encoder point of view, you may have a point.

Reply:
> From the encoder point of view, you may have a point.

I just realized, I already did it directly in the encoder. 😊